### DISCUSSION TITLE
Latest requests done

### DISCUSSION THREAD

**Discussion by rchiileea** [elax46/custom-brand-icons#542] [General]
[29-01-24.zip](https://github.com/elax46/custom-brand-icons/files/14087460/29-01-24.zip)

**Comment by rchiileea** [comment]
[11-02-24.zip](https://github.com/elax46/custom-brand-icons/files/14232389/11-02-24.zip)

**Comment by rchiileea** [comment]
@elax46

**Reply by elax46** [reply]
when you post in threads always tag me because I don't get notifications

**Comment by rchiileea** [comment]
[21-02-24.zip](https://github.com/elax46/custom-brand-icons/files/14357428/21-02-24.zip)
@elax46

**Comment by rchiileea** [comment]
@elax46 Also I noticed from the iconfy site some of icons were too small and off to the left (I tried them in my preview manager and they showed up like this, I have donwloaded the ones that are like this into a folder and everytime I do a set ill do 10 to 15 corrections on them and keep file names the same so you know which ones to change.
[260284.zip](https://github.com/elax46/custom-brand-icons/files/14410368/260284.zip)

**Comment by elax46** [comment]
> @elax46 Also I noticed from the iconfy site some of icons were too small and off to the left (I tried them in my preview manager and they showed up like this, I have donwloaded the ones that are like this into a folder and everytime I do a set ill do 10 to 15 corrections on them and keep file names the same so you know which ones to change. [260284.zip](https://github.com/elax46/custom-brand-icons/files/14410368/260284.zip)

ok I kindly ask you to separate the icons into two different zips, one for modifying and the other for adding new ones

**Comment by rchiileea** [comment]
[260284.zip](https://github.com/elax46/custom-brand-icons/files/14425356/260284.zip)
@elax46 seperated into two folders in the zip.

**Reply by elax46** [reply]
Valid for the next version they have already been added

**Comment by rchiileea** [comment]
[new-260324.zip](https://github.com/elax46/custom-brand-icons/files/14755896/new-260324.zip)
[fix-260324.zip](https://github.com/elax46/custom-brand-icons/files/14755897/fix-260324.zip)

Also you may want to look at the coding as it seems other icon and repos have adjusted theres lately ours currently looks like this

![image](https://github.com/elax46/custom-brand-icons/assets/87599807/9360a5bd-1774-43b9-8193-8e86953d55ca)15"],

were theres would look like this and might be something needed to adapt to fix some issues

![image](https://github.com/elax46/custom-brand-icons/assets/87599807/205a9de0-8a45-48bd-97b4-ff5a87f5fd74)

**Reply by elax46** [reply]
Thanks for your work as always, but I didn't understand what was changed compared to the other projects

**Comment by rchiileea** [comment]
by the looks of it, its just the spacing of the  
```
], 

```
in the java, but also if you open our js and compare to others in visual its spaced differently too on the code side, it might be nothing but I have compared to a few files now and they all our simular. ours is different

**Reply by elax46** [reply]
it's just a question of indentation it's nothing functional but just to make the code "nicer to look at"

**Reply by elax46** [reply]
@rchiileea Personal request :) in the next version could you create two small icons of the [P1S](https://www.google.it/search?sca_esv=c9db5b8f38fb0f2e&sxsrf=ACQVn093HiKO4LYR2qOHB3K19y8ElEVCqg:1711579137161&q=P1S&tbm=isch&source=lnms&prmd=ivnbz&sa=X&ved=2ahUKEwjzu-aTwZWFAxVog_0HHfscBoUQ0pQJegQIDhAB&biw=1136&bih=1274&dpr=1) and [A1 Mini ](https://www.google.it/search?q=A1+Mini&tbm=isch&ved=2ahUKEwigxJqUwZWFAxXT5gIHHRQDDNoQ2-cCegQIABAA&oq=A1+Mini&gs_lp=EgNpbWciB0ExIE1pbmkyBxAAGIAEGBMyBxAAGIAEGBMyBhAAGB4YEzIIEAAYBRgeGBMyCBAAGAUYHhgTMggQABgFGB4YEzIIEAAYBRgeGBMyCBAAGAUYHhgTMggQABgFGB4YEzIIEAAYBRgeGBNIjhZQsApYyRRwAHgAkAEAmAFXoAGKBKoBATe4AQPIAQD4AQGKAgtnd3Mtd2l6LWltZ8ICBBAjGCfCAggQABgHGB4YE8ICBRAAGIAEwgIIEAAYgAQYsQPCAgQQABgDwgIOEAAYgAQYigUYsQMYgwHCAgQQABgeiAYB&sclient=img&ei=AqAEZqA7082L6A-UhrDQDQ&bih=1274&biw=1136&prmd=ivnbz)bambulab? I need them as icons for smart sockets

**Comment by rchiileea** [comment]
@elax46 you mean something like this?
![image](https://github.com/elax46/custom-brand-icons/assets/87599807/0d9545b0-edef-4024-abf6-8926fcc915ba)
he he

**Reply by elax46** [reply]
That's great!

**Comment by rchiileea** [comment]
![image](https://github.com/elax46/custom-brand-icons/assets/87599807/352c1a2d-f6f3-48eb-83b7-177391d25a47)

**Reply by elax46** [reply]
it's great! Thank you

**Comment by rchiileea** [comment]
[fixed-1-4-24.zip](https://github.com/elax46/custom-brand-icons/files/14821515/fixed-1-4-24.zip)
[new-1-4-24.zip](https://github.com/elax46/custom-brand-icons/files/14821516/new-1-4-24.zip)

**Reply by rchiileea** [reply]
@elax46

**Reply by elax46** [reply]
`adore-alt` is multi-path

**Comment by rchiileea** [comment]
all the fixed ones are I forgot to remove the surround... doh, so expect 20 in new update of fixed icons :( doh

**Comment by rchiileea** [comment]
@elax46 see above post lol,

here they all are fixed
[fixed-1-4-24.zip](https://github.com/elax46/custom-brand-icons/files/14842771/fixed-1-4-24.zip)
Then this new batch of fixed icons

[fix-0204.zip](https://github.com/elax46/custom-brand-icons/files/14842773/fix-0204.zip)

and then new ones added, if at all possible can we get these icons tonight or in the morning so I can share tomorrow on release party (contain quite a few stairs icons for the new floors introduced in 2024.4)

[new-0204.zip](https://github.com/elax46/custom-brand-icons/files/14842785/new-0204.zip)

**Comment by rchiileea** [comment]
@elax46 
[1104.zip](https://github.com/elax46/custom-brand-icons/files/14944484/1104.zip)
[fixed-1104.zip](https://github.com/elax46/custom-brand-icons/files/14944485/fixed-1104.zip)

**Comment by rchiileea** [comment]
[toofix.zip](https://github.com/elax46/custom-brand-icons/files/15073344/toofix.zip)
last of these hopefully
[2204.zip](https://github.com/elax46/custom-brand-icons/files/15073351/2204.zip)
new requests and a few others

**Reply by elax46** [reply]
`aqara-h1-3` have a duble path @rchiileea Have you looked at the[ icons](https://elax46.github.io/custom-brand-icons/) that are still having problems here? there are so many...

**Comment by rchiileea** [comment]
[aqara-h1-3.zip](https://github.com/elax46/custom-brand-icons/files/15090068/aqara-h1-3.zip)
@elax46

Having looked up the iconify project I repaired the ones that showed off to the left in that (and in many other apps when you use iconfy as that has a way of showing faults)... if you look through the list  https://icon-sets.iconify.design/cbi/ you will see only the ones showing small and to the left upper corner and the ones remaining in the folder too fix i sent you. although I have noticed some of the ones I sent previously have not been updated  into our repo either (sockets, signee etc) 

So I will have to go through that list again once you have updated all the too fix items, there are more though I notice (but once you have added all the too fix I can find them) as hovering now shows sizes and there are quite a few showing over 24x24 so more to fix, I can not try to fix until we have all the latest ones added, then I can then go through the oversized ones and fix them.

Edit, I have noticed as well just going through the ones I think need redoing (over sized ones due to view box being between 32x32 and some even 5000x5000 view box lol... and some are mine, but others are user pull requests.

**Comment by rchiileea** [comment]
p.s get the owner of the iconify repo to add by Emanuele and rchiileea :P

**Comment by rchiileea** [comment]
[adore-alt.zip](https://github.com/elax46/custom-brand-icons/files/15090388/adore-alt.zip)
use this adore alt

**Comment by rchiileea** [comment]
@elax46 these icons in the list need changing, but luckily you only need to change view box in the file

viewbox changes in json file all changed to 0 0 24 24

sync-box
sync-box-alt
socket-us
socket-eu
socket-uk
beyond-down
beyond-up
blind-tilt-closed
blind-tilt-open
bulb-filament-alt
bulb-filament-candle
bulb-group-ceiling-round
bulb-group-ceiling-square
ceiling-buratto-two
ceiling-cher-semiflush
ceiling-explore
ceiling-fair
ceiling-fair-semiflush
ceiling-fugato-four
ceiling-fugato-four-alt
ceiling-fugato-three
ceiling-fugato-three-alt
ceiling-fugato-two
ceiling-fugato-two-alt
ceiling-muscari
ceiling-friends-of-hue-arke
ceiling-friends-of-hue-eikon
liane
lightstrip-tv
play-bar-v-alt
play-bar-two
signe
signe-gradient-floor
signe-gradient-table
amaze-alt
beyond

only view box needs changing as they sit to the left in apps using iconify and your web app, these have to be changed.

then the bad news 
214 files are have a larger viewbox but show up in centre on iconify and in home assistant, but the issue shows up in your web browser viewer at oversized too, I have tested all 214 of these icons and they work in ha as expected, if you change your web viewer app to say 32 32 viewbox this will be fine as these slightly oversized ones will be fine in the webviewer app and in the ha and iconify.

Then their should only be 80 odd that will need resizing due to the fact they range from 128 128 viewbox up too (which shcked me) 5000 5000 view box (and explains why some of the icons are not visible in your web based app at all, I expect the 32x23 ones are the ones that only show part of the icon).... but these could be a explination of why people have caching issues too.

So first correct the icons listed above to view box 0 0 24 24

do not correct any others, then change your web app view box to 0 0 32 32

Then I will go through the list again and download all the files above that to correct, once this is done If we get any caching issues of icons from users moving forward I will have to resize the ones that are 0 0 32 32... But I am hoping the HA app will accept that size and i dont have to as 50 icons to correct is much better than 214.

**Reply by elax46** [reply]
thanks for your detailed analysis, I will fix the viewboxes of the icons you indicated and check the issue on the icon site to understand if I can modify the code in relation to the icon viewbox without setting a static viewbox

**Reply by elax46** [reply]
update: I corrected all the icons you indicated, I also updated the code to [find](https://elax46.github.io/custom-brand-icons/) the icons with the viewbox and at 32x32 it seems to work for many icons but several still look bad so they need to be modified

**Reply by elax46** [reply]
icons with problems:
- smart-plug-schuko
- smart-power-strip
- smart-desk-lamp
- smart-plug-it
- temp-sensor
- double-smart-plug
- double-wall-switch
- triple-wall-switch
- single-wall-switch
- smart-switch
- lamp-bedside-table
- lamp-bedside-table
- lamp-bedside-table
- essential-oil-diffuser
- roller-shutter-switch
- arlo-essential-indoor
- arlo-cam
- arlo-pro-3
- arlo-ultra-2
- arlo-security-light
- arlo-q
- arlo-video-doorbell
- arlo-floodlight
- arlo-audio-doorbell
- arlo-base-station
- arlo-baby
- orbi
- wifi-extender
- nighthawk
- mini
- window-sensor
- usb-smart-adaptor
- smart-plug-sonoff
- bridge
- smart-camera
- battery-charged
- battery-full
- battery-good
- battery-half
- battery-low
- battery-critical
- nexxt
- fastgate
- moes-switch-single
- moes-switch-single
- moes-switch-single
- echo-dot-gen-3
- echo-gen-2
- echo-spot
- echo-plus
- echo-show-5-gen-2
- air-conditioner
- thermostat
- charging-station
- garden-irrigation
- apple-airpods
- apple-iphone
- nextcloud
- shelly-logo
- apple-ipad-pro
- sync-module
- xt2-camera
- nas-v2
- octoprint
- amazon-logo
- homepod-mini
- homepod
- apple-tv-box
- apple-watch
- google-home-icon
- linktap
- tim-logo
- ubiquiti
- salt-lamp
- ubiquiti-logo
- yamaha-rx-v773
- google-wifi-router
- dune-hd
- sonos-bookshelf-vertical
- christmas-wreath
- christmas-tree
- christmas-tree-v2
- ring-alarm-panel
- ring-floodlight
- discovery-plus
- reolink-820
- seagate-ssd-m2
- rituals
- telnet-box
- laptop-windows
- litterbox
- fritz-repeater
- fritzbox-7530
- asus-rp-68u
- sonos-arc
- lg-signiture
- lg-fridge
- lg-cooker
- lg-aircon
- lg-air-purifier
- npo
- lift-in-window-open2
- bitwarden
- outside
- driveway
- discord
- telegram

**Reply by rchiileea** [reply]
lol I am going to busy, thanks for saving me the time of going through them, I knew there would be a few.

**Reply by elax46** [reply]
take all the time you need

**Comment by rchiileea** [comment]
back from break, so i went through again my self and found 130 that need adjusting/fixing in total lol

**Comment by rchiileea** [comment]
so been busy, I had to redo a lot of these completely as they had way to many points some had 1000 points, all the arlo stuff has been redone, fixed points on some of the other stuff and redone all the battery ones, redone the apple ones too
[fix part 1.zip](https://github.com/elax46/custom-brand-icons/files/15279231/fix.part.1.zip)
55 in that folder lol sorry, ill crack on with new request begining of the week as your going to have a busy weekend with these, remember to make sure viewbox is 24 not 24000 :P

only 75 to go

**Reply by elax46** [reply]
2 hours of work! Anyway, thank you always for your support `arlo-security-light` has a dual path

**Comment by rchiileea** [comment]
latest requests and redone
[update and two fixed icons.zip](https://github.com/elax46/custom-brand-icons/files/15305548/update.and.two.fixed.icons.zip)
 arlo security is in there, so is arlo floodlight, did not like the circle on one in fixed partt 1 zip.  @elax46

**Comment by rchiileea** [comment]
@elax46 omg I have been busy in spare time, so many icons I actually redone most of them too so they had much few anchors, some that were over 1mb are now 2kb... some like the sonos obviously could not be changed due to the grill effect they have.
[fix part 2.zip](https://github.com/elax46/custom-brand-icons/files/15474745/fix.part.2.zip)
[fix part 3.zip](https://github.com/elax46/custom-brand-icons/files/15474747/fix.part.3.zip)

I then had a blast from the past redoing the moes ones as those are icons I requested after donating a long time ago now, way before i taught myself to do icons lol..
![cbi--moes-switch-single](https://github.com/elax46/custom-brand-icons/assets/87599807/948f4734-41f6-4bba-851e-0efdef544291)
![cbi--moes-switch-dual](https://github.com/elax46/custom-brand-icons/assets/87599807/8baf3814-1e05-4f49-9359-5673cb9f0ba6)
![cbi--moes-switch-triple](https://github.com/elax46/custom-brand-icons/assets/87599807/d848808b-5208-41ab-b928-57d20401197d)

new icons are will be uploaded tomorrow

hopefully this will help with caching and sizing issues for your web icon browser

**Reply by elax46** [reply]
> @elax46 omg I have been busy in spare time, so many icons I actually redone most of them too so they had much few anchors, some that were over 1mb are now 2kb... some like the sonos obviously could not be changed due to the grill effect they have. [fix part 2.zip](https://github.com/elax46/custom-brand-icons/files/15474745/fix.part.2.zip) [fix part 3.zip](https://github.com/elax46/custom-brand-icons/files/15474747/fix.part.3.zip)
> 
> I then had a blast from the past redoing the moes ones as those are icons I requested after donating a long time ago now, way before i taught myself to do icons lol.. ![cbi--moes-switch-single](https://private-user-images.githubusercontent.com/87599807/334542795-948f4734-41f6-4bba-851e-0efdef544291.svg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5MzM0NTYsIm5iZiI6MTcxNjkzMzE1NiwicGF0aCI6Ii84NzU5OTgwNy8zMzQ1NDI3OTUtOTQ4ZjQ3MzQtNDFmNi00YmJhLTg1MWUtMGVmZGVmNTQ0MjkxLnN2Zz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI4VDIxNTIzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlMzE4YjJlYjRlZDU0OGZjMjljMTY0Yzk2Zjc5OWRkMDRiMmJhNDEyN2ZhOTQ2NWQxY2QzZmQ2NzdiOTRhZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JfPJAGLvY5Hx4i9l2EiwjsNGczzj6ZDms71nwIyHjKE) ![cbi--moes-switch-dual](https://private-user-images.githubusercontent.com/87599807/334542799-8baf3814-1e05-4f49-9359-5673cb9f0ba6.svg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5MzM0NTYsIm5iZiI6MTcxNjkzMzE1NiwicGF0aCI6Ii84NzU5OTgwNy8zMzQ1NDI3OTktOGJhZjM4MTQtMWUwNS00ZjQ5LTkzNTktNTY3M2NiOWYwYmE2LnN2Zz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI4VDIxNTIzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJmN2E3ZmFhZWYwNThiYjViYTFjOGM1NTMzZWFmM2YyMWViY2YzNTAzNzZiYjI2YWU1MmNmMzliYWQ5ODBiMzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CaGVqcsiU5t2lt8Pnfx_0OkYmrunvvLDo7czOH8kfZA) ![cbi--moes-switch-triple](https://private-user-images.githubusercontent.com/87599807/334542802-d848808b-5208-41ab-b928-57d20401197d.svg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5MzM0NTYsIm5iZiI6MTcxNjkzMzE1NiwicGF0aCI6Ii84NzU5OTgwNy8zMzQ1NDI4MDItZDg0ODgwOGItNTIwOC00MWFiLWI5MjgtNTdkMjA0MDExOTdkLnN2Zz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI4VDIxNTIzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiODU5Y2NkZjg1OTY1MjQ1NTVkM2YxOGJlMGRkNDNmZWUyMGFkNTA1MGFmMGY2YTk2MGE3YmM5Nzc0ZTBmM2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.sIW11BsuGM2YbIpmmUL3k8U2qdYJ8Eukqv2m-imBzUY)
> 
> new icons are will be uploaded tomorrow
> 
> hopefully this will help with caching and sizing issues for your web icon browser

remarkable work for both

**Comment by elax46** [comment]
From the icon creator it seems that there are still some icons that are not visible @rchiileea 

- lg-cooker
- lg-cooker
- lg-fridge
- laptop-windows
- lift-in-window-open2

**Comment by rchiileea** [comment]
[last fixes.zip](https://github.com/user-attachments/files/15517058/last.fixes.zip)
@elax46

**Comment by rchiileea** [comment]
@elax46  is everything alright bud are you ok?

**Reply by elax46** [reply]
Yes, everything is ok. I didn't release the version of the fixed icons because I was waiting for the new ones

**Comment by rchiileea** [comment]
Fix may 14 post had most of the new ones in 🤣

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by rchiileea** [comment]
[update.and.two.fixed.icons.zip](https://github.com/user-attachments/files/15945111/update.and.two.fixed.icons.zip)
This one

**Reply by elax46** [reply]
@rchiileea 
I think there are only two icons left to fix
- `phu:lamp-bedside-table`
- `phu:lamp-bedside-table-2`

**Comment by rchiileea** [comment]
my bad I thought I had uploaded all the new icons :( added the two new requests but created a generic battery back up, also two icons left to fix are in there redesigned.
[24062024.zip](https://github.com/user-attachments/files/15954199/24062024.zip)

**Comment by rchiileea** [comment]
@elax46 added new icon requests as well as a few others including smart camera motion (car/person/pet) icons

[150724.zip](https://github.com/user-attachments/files/16232950/150724.zip)

**Comment by rchiileea** [comment]
@elax46 
[03-09-24.zip](https://github.com/user-attachments/files/16847493/03-09-24.zip)
new icons done including the lidl lamp I commented, I have also updated the readme for the template too

**Reply by elax46** [reply]
thanks for your work!

**Comment by rchiileea** [comment]
@elax46 sorry big update for you :( all the latest requests done plus the adition of the boca junior one in pull request done correctly, I have also added 13 of the twinkly brand lights as well as the logo (14 in total)
[220924.zip](https://github.com/user-attachments/files/17100368/220924.zip)

edit forgot the panasonic icon @elax46 
[panasonic.zip](https://github.com/user-attachments/files/17100531/panasonic.zip)

**Reply by elax46** [reply]
The twinkly-festoon and twinkly-c9 icons have a double path @rchiileea

**Reply by rchiileea** [reply]
@elax46 fixed
[New folder.zip](https://github.com/user-attachments/files/17111463/New.folder.zip)

**Comment by rchiileea** [comment]
[081024.zip](https://github.com/user-attachments/files/17290408/081024.zip)
@elax46 Latest icons done including the pull request ones so delete the pull request

**Comment by rchiileea** [comment]
@elax46 some are replacements (updated logos from brands) jellyseer, robin hood, deezer

[16112025.zip](https://github.com/user-attachments/files/17784158/16112025.zip)

**Reply by elax46** [reply]
> @elax46 some are replacements (updated logos from brands) jellyseer, robin hood, deezer
> 
> [16112025.zip](https://github.com/user-attachments/files/17784158/16112025.zip)

kallax 8 has a double path

**Comment by rchiileea** [comment]
[ikea-kallax8.zip](https://github.com/user-attachments/files/17791941/ikea-kallax8.zip)
@elax46

**Reply by elax46** [reply]
Hi, is everything ok? It's been a while since we've heard from each other? @rchiileea

**Reply by rchiileea** [reply]
I have a 9 year ild son, the project I was working on and the new year, also currently building extension to the house so free time is limited at the moment with all spare time used for work to pay for everything.

**Comment by rchiileea** [comment]
only another 6 to go :P 
![image](https://github.com/user-attachments/assets/8ed5111e-5031-41ed-abbd-033197e7de93)

**Comment by rchiileea** [comment]
@elax46 enjoy, i know you wont with this many as  I did not on this one so many hours not enough time, cleared all but 1 i think, but 58 icons phew.

[done.zip](https://github.com/user-attachments/files/18551218/done.zip)

**Reply by elax46** [reply]
thank you so much for your incredible work considering the little time between work and family! now it's my turn!

**Comment by rchiileea** [comment]
@elax46 quickly knocked out the window ones too

[27-1-25.zip](https://github.com/user-attachments/files/18555824/27-1-25.zip)

**Comment by rchiileea** [comment]
[09-02-25.zip](https://github.com/user-attachments/files/18732137/09-02-25.zip)
@elax46

**Comment by rchiileea** [comment]
[may.zip](https://github.com/user-attachments/files/20389843/may.zip)
2 have not been done yet due to doing quite a few hand drawn ones, blusound and sonos 5, left comment that they will be done next round. @elax46

**Reply by elax46** [reply]
Thanks for your great effort as always I will integrate everything as soon as possible, thanks

**Comment by rchiileea** [comment]
[newicons.zip](https://github.com/user-attachments/files/20522669/newicons.zip)
@elax46 This one please forgot to add f1-alt

**Comment by rchiileea** [comment]
[june2.zip](https://github.com/user-attachments/files/20693968/june2.zip)
all done and added 95% of the nanoleaf products @elax46

**Comment by rchiileea** [comment]
[lampada-parete.zip](https://github.com/user-attachments/files/21429490/lampada-parete.zip)
@elax46

**Comment by rchiileea** [comment]
hey as you may have noticed I have less time to do icons due to work commitments, but I am going to pull away from requests that ask for multiple hand drawn icons but dont share no love, all this work, i have one permanment sponsor and 2 single donations my time is precious at the moment due to cost of doing home etc and currently to coslty to spend hours hand drawing the harder requests as well as ones that are 5 or 6 at once without a coffee or a thanks @elax46

**Reply by elax46** [reply]
Don't worry, there's no problem, obviously there are other priorities

**Reply by elax46** [reply]
Hi @rchiileea , I wanted to let you know about some new developments regarding the project.

-  I've updated the icon finder to allow users to select icons and download a stripped-down version of the js file to try to limit display issues.
- I'm working on a fully automated version of the js file creation and its release to GitHub. Simply place the icons in the `icon-svg` folder. The Node script and GitHub workflows will take care of the rest.

**Comment by rchiileea** [comment]
thank you, I will be back with a bang soon, so many icons to do, just have not had the time.

**Reply by elax46** [reply]
Well, great news! - [CBI is now fully automated!](https://github.com/elax46/custom-brand-icons/releases/tag/2025.12.0b)

**Comment by elax46** [comment]
@rchiileea Hi, how are you? I hope we can resume support for the project.

**Comment by rchiileea** [comment]
sorry guys, but I am currently super busy with a project with intel/tamiya/bambu labs for completion in buy end of may... please bear with me... house is finished at least... and still not getting git hub notifications no more.... mmm, need to email them.
